### ISSUE TITLE
Add list of member organizations to website.

### ISSUE THREAD

**Issue by krusynth** [civictechnologists/website#7]
Probably should have its own page, maybe a more fully-built-out about page?  Also a summary on the home page to lend credibility (since you're asking for donations and all).

**Comment by bensheldon** [comment]
Agreed 👍🏻 Membership is developing so I'd like to decouple the design and implementation from the data. Let's do the website changes in a branch/PR with fake data, and then we'll publish those changes once we have the real membership data. That means we might have to be somewhat conservative initially with the design to keep it contained and avoid merge-conflicts.

I like the suggestion of creating an About page to contain this content. That satisfies my requirement that we keep the changes self-contained. We're in a content-building phase right now, so I expect that we'll reorganize pages and navigation later.

**Comment by krusynth** [comment]
We can use data files to separate the content from the layout.  See here:  https://jekyllrb.com/docs/datafiles/

Examples:

https://github.com/krusynth/digitalpolicy/tree/ghpages/_data/acronyms
https://github.com/krusynth/billhunt.dev/tree/www/_data

You can also do this as front matter yaml in the page itself, but in my experience that is painful to maintain and I don't recommend it. (E.g. https://raw.githubusercontent.com/krusynth/billhunt.dev/www/about.md )

**Comment by bensheldon** [comment]
My preference would be for using data files 👍 Let's be as Jekylly as possible.

**Comment by bensheldon** [comment]
We're just waiting on the member lists now.

**Comment by GovInTrenches** [comment]
Opening this back up! 

Current (or at least soon to be) Members: 
Civic Tech Atlanta	Atlanta	GA	https://www.civictechatlanta.org	https://www.meetup.com/civictechatlanta/
Code For Dayton	Dayton	OH	codefordayton.org	https://www.meetup.com/gem-city-tech/
SF Civic Tech	San Francisco, Remote	CA	https://www.sfcivictech.org/	https://www.meetup.com/sfcivictech/
Code for Philly	Philadelphia	Pennsylvania	https://codeforphilly.org	https://www.meetup.com/code-for-philly/
BetaNYC	New York City	NY	www.beta.nyc	https://beta.nyc/events/
Chi Hack Night	Chicago	IL	https://chihacknight.org/	https://chihacknight.org/events/index.html
Civic Tech New England	Boston	MA	http://codeforboston.org/	https://www.meetup.com/code-for-boston/
Code for BTV	Burlington	VT	codeforbtv.org	https://www.meetup.com/codeforbtv/

My _ideal_ would be a map of where everyone is -- but for now I think we can just list members and link them to their site. Open to how we handle this part since we'll have to (hopefully) scale to a longer list. I do like how Code for All does this too https://codeforall.org/our-global-network

**Comment by krusynth** [comment]
You should just need to update these files:  https://github.com/krusynth/act-website/commit/bf6e0d17113bb839f836c0e99455751a7cc7e87c#diff-d292f13f8d21bbaa7580dcbc524afeb4c63a68664ec8dff71fedd56242ab5a55

I'm afraid I can't help you with the maps, probably easiest just to draw one and have it be non-interactive anyway.
